newproject01/newproject01/q1.c

```diff
@@ -0,0 +1,67 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#define N 8
+
+isNumberInOrder(int[], int, int); /*Declaration of a function*/
+
+int main1()
+{
+	int arr[N], number, i, flag = 1, counter = 0;
+
+	while (flag)
+	{
+		printf("Enter %d integer numbers, please: ", N);
+
+		for (i = 0; i < N && flag; i++)
+			if (scanf("%d", &arr[i]) != 1)
```

Comment:
Why does scanf return value shouldn't be 1?
also, define a const for 1 and give it a meaningful name.